tests/client/amo/containers/TestSearch.js

```diff
@@ -170,14 +167,10 @@ describe('CurrentSearchPage.loadSearchResultsIfNeeded()', () => {
       .returns(Promise.resolve({ entities, result }));
     return loadSearchResultsIfNeeded({ store, location }).then(() => {
       mockApi.verify();
-      assert(
-        dispatch.firstCall.calledWith(
-          searchActions.searchStart({ filters, page })),
-          'searchStart not called');
-      assert(
-        dispatch.secondCall.calledWith(
-          searchActions.searchLoad({ filters, entities, result })),
-          'searchLoad not called');
+      expect(dispatch.firstCall.calledWith(
+        searchActions.searchStart({ filters, page }))).toBeTruthy();
```

Comment:
mmm, I want to kill all `calledWith()` assertions one day. The failure message is never helpful. Anyway, that's for later.